src/ProxyClient/Varnish.php

```diff
@@ -36,6 +36,15 @@ class Varnish extends HttpProxyClient implements BanCapable, PurgeCapable, Refre
     const HTTP_HEADER_HOST = 'X-Host';
     const HTTP_HEADER_URL = 'X-Url';
     const HTTP_HEADER_CONTENT_TYPE = 'X-Content-Type';
+
+    /**
+     * Default name of the header used to invalidate content with specific tags.
+     *
+     * This happens to be the same as TagHeaderFormatter::DEFAULT_HEADER_NAME
+     * but does not technically need to be the same.
+     *
+     * @var string
+     */
     const DEFAULT_HTTP_HEADER_CACHE_TAGS = 'X-Cache-Tags';
```

Comment:
Just a note here: X-headers are deprecated ([RFC 6648](https://tools.ietf.org/html/rfc6648)), not sure if you would like to adjust them while releasing `2.0`.